Tests/SentryTests/SentryClientTests.swift

```diff
@@ -315,6 +315,23 @@ class SentryClientTest: XCTestCase {
         }
     }
     
+    func testCaptureEventWithCurrentScreenInTheScope() {
+        SentryDependencyContainer.sharedInstance().application = TestSentryUIApplication()
+        
+        let event = Event()
+        event.exceptions = [ Exception(value: "", type: "")]
+        
+        let scope = fixture.scope
+        scope.currentScreen = "TestScreen"
+        
+        fixture.getSut().capture(event: event, scope: scope)
+        
+        try? assertLastSentEventWithAttachment { event in
+            let viewName = event.context?["app"]?["view_names"] as? [String]
+            XCTAssertEqual(viewName?.first, "TestScreen")
```

Comment:
I'm going to open a separate PR to refactor assertLastSentEventWithAttachment so that it's not `throws`, but generally we shouldn't be using `try?` or `try!` in test cases. I understand it seems pointless in this case, and it is in 99% of calls to assertLastSentEventWithAttachment.

Reply:
here it is: https://github.com/getsentry/sentry-cocoa/pull/4114